server/libs/database.js

```diff
@@ -188,12 +185,10 @@ module.exports.mapJsonToColumns = function(jsonObj) {
   mapping += jsonObj['Battery']['HighCellVoltage'] + ', ';
   mapping += jsonObj['Battery']['HighCellVoltageId'] + ', ';
   mapping += jsonObj['Battery']['AverageCellVoltage'] + ', ';
-  // Ignore PrechargeState until Hermes is updated
-  // and replace with Off instead
-  // mapping += jsonObj['Battery']['PrechargeState'] + ', ';
+  // mapping += jsonObj['AuxBms']['PrechargeState'] + ', ';
```

Comment:
Does this still need to be fixed?

Reply:
I just pulled the latest changes for the test tool and they're still sending the wrong values. The documentation says that it must be one of: `'Off', 'Common Engaged', 'Charge Engaged', 'Discharge Engaged', 'All Engaged'`.

Currently it is sending `'Enable Pack'`